src/content/docs/ai-monitoring/explore-ai-data/view-model-data.mdx

```diff
@@ -0,0 +1,171 @@
+* <DoNotTranslate>**Response errors by model**</DoNotTranslate>: Determine if one specific model produces more errors on average, or if one specific error is occurring across your models.
+* <DoNotTranslate>**Response errors by type**</DoNotTranslate>: View how often certain errors appear. 
+* <DoNotTranslate>**Errors table**</DoNotTranslate>: View error type and message in context of the request and response. 
+
+### Performance tab [#performance-inventory]
+
+<img
+    title="Model inventory: Performance"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryPerformancePage}
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Performance</DoNotTranslate>**: View data about your AI model's performance.
+</figcaption>
+
+The performance tab aggregates response and request metrics across all your models. Overview the models that take the most time to process a request or create a response with the pie charts, or refer to the time series graphs to track upticks in request or response times. 
+
+* The 95th percentile, or **p95**, indicates the number of transactions that were below a certain value during a time period. If you've filtered your time series to p95 and see that a call at 9:00pm took six seconds to complete, it means that 95% of all calls were below six seconds, but 5% of calls were above six seconds.
+* The 99th percentile, or **p99**, indicates that 99% of calls or transactions were below a given value during a time period. Inversely, it means 1% of transactions were above the value. 
+
+You can use performance charts to locate outliers across your models. 
+
+### Cost tab [#cost-inventory]
+
+<img
+    title="Model inventory: Performance"
+    alt="A screenshot of the Errors time series and chart"
+    src={aiModelInventoryCostPage}
+/>
+<figcaption>
+    Go to **<DoNotTranslate>[one.newrelic.com](https://one.newrelic.com) > All Capabilities > AI monitoring > Model inventory > Cost</DoNotTranslate>**: View data about your AI model's cost.
+</figcaption>
+
+The cost page uses a combination of time series graphs and pie charts to identify cost drivers amongst your models. Determine whether an increase in cost came from prompts or completions, or if certain models cost more on average than others.  
+
+* <DoNotTranslate>**Tokens used and token limit**</DoNotTranslate>: Evaluate how often your models approach a given token limit.
+* <DoNotTranslate>**Total tokens by models**</DoNotTranslate>: Determine which of your models use the most tokens on average.
+* <DoNotTranslate>**Total usage by prompt and completion tokens**</DoNotTranslate>: Understand what ratio of tokens comes from prompts your model accepts against tokens used per completion. 
+
+When you can see how your tokens are used, you can improve how your AI app uses one or more of your models when it generates its final response to an end user. 
```

Comment:
This is a passive sentence. I'd write, for example, this: `When you see how you use your tokens`